core/server/adapters/scheduling/SchedulingDefault.js

```diff
@@ -53,19 +53,20 @@ SchedulingDefault.prototype.unschedule = function (object) {
  * because allJobs is a sorted list, we don't have to iterate over all jobs, just until the offset is too big
  */
 SchedulingDefault.prototype.run = function () {
-    var self = this,
-        timeout = null,
+    const self = this;
+    let {isRunning} = this;
```

Comment:
🤔 What is the reason you are not accessing `this.isRunning`?

Reply:
I tried to completely refactor out `self`, but some integration tests started failing. Therefore, I left `self` in for now. I'd like to revisit it in the future, but I didn't want that to stop from me submitting what I had done so far.

Reply:
I don't see a reason to change `this.running`. This is IMO independent from the ES6 task/issue. It just changes the coding style. Could you please revert? Thanks!

Reply:
I will revert it.

Reply:
It's been reverted.
https://github.com/TryGhost/Ghost/pull/9696/commits/91239fd731e9d0278c08e9515f5112b774f2dd5a